client/v3/config.go

```diff
@@ -66,6 +66,9 @@ type Config struct {
 	// Password is a password for authentication.
 	Password string `json:"password"`
 
+	// Token is a JWT used for authentication instead of a password.
+	Token string `json:"token"`
```

Comment:
Given that this will be exported and it may be difficult to change the name later on, I feel like "Token" may be too generic. Would "JWT" be a better name (more implementation specific)?

Reply:
Theoretically, it can be any token types that are supported by etcd. The etcdserver just delegates to the related token prover to parse the token based on the token type configured in `--auth-token`.

But in practice, it's JWT for now. So suggest to keep using `Token`